samples/bluetooth/mesh/printy_timey_blinkey/src/main.c

```diff
@@ -0,0 +1,7 @@
+#include <zephyr.h>
+#include <sys/printk.h>
+
+void main(void)
+{
+	printk("Hello WUrld! %s\n", CONFIG_BOARD);
```

Comment:
Are you sure it's not `Hello World`
```suggestion
	printk("Hello World! %s\n", CONFIG_BOARD);
```